api/db-cdr/generate-cdr/build-prep-snomed-rel-meas.sh

```diff
@@ -58,8 +58,8 @@ WHERE concept_id in
             and b.standard_concept = 'S'
     )"
 
-# currently, there are only 4 levels, but we run it 5 times to be safe
-for i in {1..4};
+# currently, there are only 6 levels, but we run it 7 times to be safe
+for i in {1..6};
```

Comment:
Should this be 
for i in {1..7}; as the comment says? the comment and action are not consistent for this line

Reply:
The for loop starts at 0, so this comment is correct but the others aren't. I will update the others to be consistent.

Reply:
Good catch, fixing.